postprocessing/pp-binary.c

```diff
@@ -0,0 +1,92 @@
+/*! \file    pp-binary.c
+ * \author   Lorenzo Miniero <lorenzo@meetecho.com>
+ * \copyright GNU General Public License v3
+ * \brief    Post-processing to generate binary files out of binary data recordings
+ * \details  Implementation of the post-processing code needed to
+ * generate binary files out of binary data recordings: more precisely,
+ * the code simply extracts the data from the packets and appends it to
+ * the provided file exactly as it is, with no header/footer.
+ *
+ * \ingroup postprocessing
+ * \ref postprocessing
+ */
+
+#include <arpa/inet.h>
+#ifdef __MACH__
+#include <machine/endian.h>
+#else
+#include <endian.h>
+#endif
+#include <inttypes.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "pp-binary.h"
+#include "../debug.h"
+
+
+FILE *binary_file = NULL;
```

Comment:
If requests happen in parallel, won't they clobber this? Maybe put this in some user_data or thread local storage?

Reply:
This is part of the postprocessor, not Janus, so there's no requests to handle: each time the postprocessor runs it only handles a single file.